src/fireblocks-sdk.ts

```diff
@@ -1326,6 +1326,22 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest(`/v1/management/api-users/${id}/whitelist-ip-addresses`);
     }
 
+    /**
+     * Get the tenant's OTA (One-Time-Address) configuration
+     */
+    public async getOtaConfiguration(): Promise<{enabled: boolean}> {
+        return await this.apiClient.issueGetRequest("/v1/management/ota");
+    }
+
+    /**
+     * Update the tenant's OTA (One-Time-Address) configuration
+     * @param enable
+     */
+    public async updateOtaConfiguration(enable: boolean): Promise<{enabled: boolean}> {
```

Comment:
Amit Hochma needs to respond whether the response should be `{ enabled: boolean }` or something else.